electricpy/__init__.py

```diff
@@ -4782,6 +4782,102 @@ def wireresistance(length=None,diameter=None,rho=16.8*10**-9,R=None):
     elif R != None and length != None:
         A = rho*length/R
         return _np.sqrt(4*A/pi)
+
+def parallel_plate_capacitance(A=None, d=None, e=e0, C=None):
+    r"""
+    Parallel-Plate Capacitance Calculator.
+
+    Enter three values to calculate the remaing one. Even though every variable
+    is unitless, please use the International System of Units.
+
+    .. math:: C = \frac{\varepsilon \cdot A}{d}
+
+    Parameters
+    ----------
+    A:  float, optional
+        Area of the plate, unitless.
+    d:  float, optional
+        Distance between the plates, unitless.
+    e:  float, optional
+        Permitivity of the dielectric, unitless.
+        Default value is the permittivity of free space: 8.854E-12
+    C:  float, optional
+        Capacitance, unitless.
+    Returns
```

Comment:
pydocstyle pointed out an inconsistency here; we should have an extra blank line prior to the "Return" section as highlighted by these errors:

```
./__init__.py:4787 in public function `parallel_plate_capacitance`:
        D4[10](https://github.com/engineerjoe440/ElectricPy/runs/5686294364?check_suite_focus=true#step:5:10): Missing blank line after section ('Parameters')
./__init__.py:4787 in public function `parallel_plate_capacitance`:
        D4[11](https://github.com/engineerjoe440/ElectricPy/runs/5686294364?check_suite_focus=true#step:5:11): Missing blank line before section ('Returns')
Error: Process completed with exit code 1.
```